modules/static/bufbuild/protovalidate-testing/initref

```diff
@@ -0,0 +1 @@
+v0.1.3
```

Comment:
Nit: files in the repo have a final line, probably except for the `sync` directory.

Reply:
I could have sworn I configured vscode to do always add these... 😞